application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -1482,6 +1482,8 @@ type QueryType {
     triangle: InputTriangle,
     "List of routes and agencies which are given lower preference when planning the itinerary"
     unpreferred: InputUnpreferred,
+    "The list of points the journey is required to pass through."
```

Comment:
There are some mentions of "journey" in the descriptions here. Should we replace those with itinerary?

Reply:
Additionally, the transmodel schema mentions on this field that "All locations are visited in the order they are listed." which I think is useful information.